ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html

```diff
@@ -88,6 +88,23 @@
             }
         },
 
+        "saslWrapping": {
+            "prototype": "blue-shark/ui/field-select.reel",
+            "properties": {
+                "element": {"#": "saslWrapping"},
+                "label": "SASL Wrapping",
+                "options": [
```

Comment:
looks like you could require the enum `LdapDirectoryParamsSaslWrapping` within the controller and bind the `options` property of the select component to the `members` property of the enum.